src/Routes.js

```diff
@@ -53,6 +55,7 @@ export const Routes = () => {
   const edgeParityInventoryListEnabled = useFeatureFlag(
     'edgeParity.inventory-list'
   );
+  const edgeGroupsUse = useEdgeGroups();
```

Comment:
I am afraid not using any isLoading flag or similar will result in a bad UX. Imagine the following scenario: user visits /groups and `InventoryGroups` component will be _always_ loaded first since the flag is false. Then, hook does the request and evaluates to true - this makes the view re-render and switch to `EdgeGroupsView` instead.

I propose to also return `isLoading` from the hook indicating whether we can finally use the flag and render the correct view.

Reply:
Another problem: this hook and also the request will be always made on any view visited in Inventory. `src/Routes.js` is not the best place to put this logic and I suggest to granulate composition and move this logic to a separate component, not to keep it in `Routes`.

Reply:
@gkarat the hook will change to true only if we log in to system with special customer , and not with new request

Reply:
@mgold1234, network request is required to be complete first before the hook will change the return value: https://github.com/RedHatInsights/insights-inventory-frontend/pull/2086/files#diff-0f486407b67781cb20ec63cea192aea4cdb84b5bbf0dec3b84feb56ee6ba4f28R16. Thus, the whole Routes component will be triggered to re-render.

Reply:
I added the isLoading , do you think that the correct place is in new component instead of Route